### ISSUE TITLE
RFP - Onetastic

### ISSUE THREAD

**Issue by SZanlongo** [chocolatey/chocolatey-package-requests#244]
Multi-purpose addin for Microsoft OneNote, including macros and calendar.

https://getonetastic.com/

**Comment by AdmiringWorm** [comment]
Changing to RFM, a package was submitted over a year ago, but no stable version was ever pushed.
https://chocolatey.org/packages/onetastic

**Comment by pauby** [comment]
I had looked at this before but was unable to get a direct download URL so
I abandoned it.

However I will pick this up again.

**Comment by AdmiringWorm** [comment]
@pauby you plan to start working on it now?
If so, we should change the status labels, otherwise it's still up for grabs until you do.

**Comment by pauby** [comment]
That's fine. Leave them as is and when I get the time to work on it I'll
change them if it's still available.

**Comment by AdmiringWorm** [comment]
Had a quick look at this one.
And looks like it's impossible to get a direct download link for the files, unless the request is a `POST` http method and the agree checkbox data is sent as well.

As such I'm adding the `Blocked Upstream` label, as it would require to get redistribution permissions from the authors before it can be made a chocolatey package.

**Comment by pauby** [comment]
I've contacted the Onetastic sales team and linked them to this thread. Hopefully we can get something put up for the community as Onetastic is a great product which I use myself! If it was part of Chocolatey thought it's much more straightforward to install and keep up to date. 

Fingers crossed.

**Comment by pauby** [comment]
A quick update on this - the team at Onetastic have got in touch and things are looking positive. Will come back once we've ironed everything out. 🤞

**Comment by gep13** [comment]
@pauby that is great news!

**Comment by pauby** [comment]
I've submitted a package for Onetastic just now. It's not the easiest installer to work with but it's done and working. Will close this once / if it's published.

**Comment by gep13** [comment]
@pauby are the Onetastic team taking ownership of the package, and pushing it when they release a new version, or are you going to be taking care of it?

**Comment by pauby** [comment]
@gep13 No, it's all taken care of by me. The software version is embedded in the installer file properties so it's a matter of pulling that down and checking.
